packages/react/src/components/Storage/File-Uploader/UploaderButton/__tests__/UploaderButton.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+
+import { UploaderButton } from '..';
+
+describe('Uploader Button', () => {
+  it('exists', () => {
+    const { container } = render(
+      <UploaderButton
+        multiple={false}
+        acceptedFileTypes={['.png']}
+        onUpload={() => ''}
+      />
+    );
+
+    expect(container).toBeTruthy();
```

Comment:
NIT: I think we prefer snapshot tests if possible to catch any future visual regressions.

Reply:
OK I'll add more tests at the end.